src/app/video-games/listing/components/video-game-listing/video-game-listing.component.html

```diff
@@ -1,11 +1,13 @@
-<ul *ngIf="videoGames.length > 0">
-  <li *ngFor="let videoGame of videoGames"
-      [routerLink]="['/videoGames', videoGame.id]">
-    <app-video-game-list-item [videoGame]="videoGame">
-    </app-video-game-list-item>
-  </li>
-</ul>
+<div class="container" *ngIf="videoGames.length > 0">
```

Comment:
This looks like a case of `div-itis`. We want to avoid div's like the plague :) 

Feel free to read on the topic - it's turns out that it's really important to angular front-end engineering/design

https://www.google.ca/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=divitis&*